### PR TITLE
Elementor page builder compatibility file

### PR DESCRIPTION
**Pull request by nicomollet**
The Elementor page builder generated pages don't work with WP Rocket (Minify mode activated). 
https://elementor.com/
There is a free and pro version.

I had to exclude every asset files like:
http://domain/wp-content/plugins/elementor/assets/*

### CODE REVIEW

File: inc/3rd-party/page-builder/elementor.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+defined( 'ABSPATH' ) or die( 'Cheatin\' uh?' );
+
+if ( defined( 'ELEMENTOR_VERSION' ) ) :
+
+    /**
+     * Excludes Elementor scripts from JS minification
+     *
+     * @since 2.9.x
```
Comment (Tabrisrp) [inline inc/3rd-party/page-builder/elementor.php]: You can set this to 2.9.5

**Comment by nicomollet** [comment]
I made the change, @since 2.9.5.

Thank you.

**Comment by Tabrisrp** [comment]
Reverted because it was merged in the wrong branch

**Comment by Tabrisrp** [comment]
Re-added code on this pull request https://github.com/wp-media/wp-rocket/pull/321

**Pull request merged**